code/modules/mob/living/simple_animal/spawner.dm

```diff
@@ -30,10 +30,9 @@
 	spawned_mobs = null
 	return ..()
 
-/mob/living/simple_animal/hostile/spawner/Life()
-	..()
-	if(!stat)
```

Comment:
It doesn't matter, since stat is checked before `handle_automated_action()` is called